src/components/cards/StakeCard.tsx

```diff
@@ -20,39 +21,42 @@ const StakeCard = (props: StakeCardProps) => {
       textAlign="center"
       {...props}
     >
-      <Flex>
-        <Text
-          fontSize={{ base: 'xs', md: 'sm', lg: 'md' }}
-          color="tooltipColor"
-          fontWeight="medium"
-        >
-          {title}
-        </Text>
+      <Text
+        fontSize={{ base: 'sm', md: 'sm', lg: 'md' }}
```

Comment:
Better written this way.
```suggestion
        fontSize={{ base: 'sm', lg: 'md' }}
```